test/gurka/test_time_tracking.cc

```diff
@@ -0,0 +1,116 @@
+#include "baldr/graphreader.h"
+#include "baldr/location.h"
+#include "gurka.h"
+#include "loki/search.h"
+#include "proto/api.pb.h"
+#include "sif/costfactory.h"
+#include "thor/pathalgorithm.h"
+
+#include <gtest/gtest.h>
+
+using namespace valhalla;
+namespace dt = valhalla::baldr::DateTime;
+
+TEST(TimeTracking, make) {
```

Comment:
probably didnt really need a gurka test to do this but it was fun to do an "integration" style test with it. so i make a trivial map. use loki to find an edge, and then use that edge to make a `Location` which we can send to the TimeInfo struct as  all the algorithms do